src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
Ensure that the `table` element exists in the DOM before proceeding. If the `table` is not found, `table` will be `null`, leading to errors.